drools-wb-services/drools-wb-verifier/drools-wb-verifier-api/src/main/java/org/drools/workbench/services/verifier/api/client/util/PortablePreconditions.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.drools.workbench.services.verifier.api.client.util;
+
+public class PortablePreconditions {
```

Comment:
Going with the PortablePreconditions in Uberfire, there is one in drools, but have to pick between drools-compiler and CDI. Not sure if CDI has any issues in the web worker, but if not then it might be useful someday.
Not worried about CDI breaking the web worker. If it does, then it is pretty easy to notice it. Everything just stops working. ;-)